addon/integrations/bing.js

```diff
@@ -57,14 +57,16 @@ export default Base.extend({
     assert('You must pass a valid `id` to the Bing adapter', id);
 
     if (canUseDOM) {
-      /* eslint-disable */
-      (function(w,d,t,r,u){var f,n,i;w[u]=w[u]||[],f=function(){
-        var o={ti:id};o.q=w[u],w[u]=new UET(o),w[u].push('pageLoad')},
-        n=d.createElement(t),n.src=r,n.async=1,n.onload=n.onreadystatechange=function(){
-          var s=this.readyState;s&&s!=='loaded'&&s!=='complete'||(f(),n.onload=n.onreadystatechange=null)
-        },i=d.getElementsByTagName(t)[0],i.parentNode.insertBefore(n,i)
-      })(window,document,'script','//bat.bing.com/bat.js','uetq');
-      /* eslint-enable */
+      if (!window.uetq) {
```

Comment:
Can I make a suggestion that these extra checks are moved into the above `ifz statements

```javascript
if (canUseDOM && !window.uetq) {
  ...
}
```

just to make things a little flatter